### PR TITLE
Remove Old Send Screen to prepare for new

### PR DESCRIPTION
**Pull request by youngkidwarrior**

### CODE REVIEW

**Comment by youngkidwarrior** [comment]
* **#382** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/382?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#381** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/381?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#380** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/380?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#379** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/379?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `dev`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @youngkidwarrior and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/app/features/account/rewards/__snapshots__/screen.test.tsx.snap
Comment (0xBigBoss) [inline packages/app/features/account/rewards/__snapshots__/screen.test.tsx.snap:1]: lol we gotta fix this test to use use fake timers. all good. I can get to it later.

File: packages/app/features/send/index.tsx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { YStack } from '@my/ui'
+import { SchemaForm, formFields } from 'app/utils/SchemaForm'
+import { FormProvider, useForm } from 'react-hook-form'
+import { z } from 'zod'
+
+export const SendScreen = () => {
+  return <SendFor />
```
Comment (0xBigBoss) [inline packages/app/features/send/index.tsx:7]: typo? also no import?

File: packages/app/features/account/rewards/__snapshots__/screen.test.tsx.snap
Comment (youngkidwarrior) [reply packages/app/features/account/rewards/__snapshots__/screen.test.tsx.snap:1]: haha yeah

Comment (0xBigBoss) [inline packages/app/features/account/rewards/__snapshots__/screen.test.tsx.snap:1]: probably need to rebase

**Review by 0xBigBoss — APPROVED** [review]

**Comment by youngkidwarrior** [comment]
### Merge activity

* **May 11, 5:36 AM EDT**: @youngkidwarrior started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/0xsend/sendapp/379).

**Pull request merged**